core/src/main/java/jenkins/security/ResourceDomainRootAction.java

```diff
@@ -277,8 +282,8 @@ private String encode() {
         }
 
         private static Token decode(String value) {
-            byte[] byteValue = Base64.getUrlDecoder().decode(value);
```

Comment:
Not strictly related, but @jsoref could make this throw an exception, so just move it down.